server/src/test/java/org/opensearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -2136,6 +2138,17 @@ public void onFailure(final Exception e) {
                         indexNameExpressionResolver
                     )
                 );
+                actions.put(
```

Comment:
this is just registering the action. Where is the TransportGetSnapshotsAction triggered? If the tests are missing for this action, please add them.

Reply:
Ack. Adding a separate test class to track the Transport actions.